### PR TITLE
chore(j-s): Rename Slack events to accommodate S-cases

### PR DESCRIPTION
**Pull request by oddsson**
# Rename Slack events to accommodate S-cases

[Asana](https://app.asana.com/0/1199153462262248/1202978837144007/f)

## What

Rename Slack events to accommodate S-cases

## Why

In R-Cases we used to talk about "Krafa". In S-cases it's "Ákæra". We are trying to use more case-agnostic words.

### Slack notifications for S-cases

Screenshot 2022-11-09 at 08 59 08

### Slack notifications for R-cases

Screenshot 2022-11-09 at 09 33 24

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/src/app/modules/event/event.service.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import {
 
 import { environment } from '../../../environments'
 import { Case } from '../case'
+import { CaseType, isIndictmentCase } from '@island.is/judicial-system/types'
```
Comment (gudjong) [inline apps/judicial-system/backend/src/app/modules/event/event.service.ts:15]: Please remove CaseType

File: apps/judicial-system/backend/src/app/modules/court/court.service.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ export const subTypes: SubTypes = {
   MAJOR_ASSAULT: 'Líkamsárás - meiriháttar',
   MINOR_ASSAULT: 'Líkamsárás - minniháttar',
   AGGRAVATED_ASSAULT: 'Líkamsáras - sérlega hættuleg',
```
Comment (gudjong) [inline apps/judicial-system/backend/src/app/modules/court/court.service.ts:46]: Please fix this as well. I checked the Auður API and in all cases the correct text is "Líkamsárás...".

File: apps/judicial-system/backend/src/app/modules/court/court.service.ts [hunk 3]
```diff
@@ -44,7 +44,7 @@ export const subTypes: SubTypes = {
...
-  ASSAULT_LEADING_TO_DEATH: 'Líkamsáras sem leiðir til dauða',
+  ASSAULT_LEADING_TO_DEATH: 'Líkamsárás sem leiðir til dauða',
```
Comment (gudjong) [inline apps/judicial-system/backend/src/app/modules/court/court.service.ts:47]: Well spotted 👍

File: apps/judicial-system/backend/src/app/modules/event/event.service.ts [hunk 4]
```diff
@@ -32,16 +33,18 @@ const errorEmojis = [
...
+  OPEN_INDICTMENT: ':unlock: Ákæra opnuð fyrir dómstól',
+  SUBMIT: ':mailbox_with_mail: Sent',
+  RESUBMIT: ':mailbox_with_mail: Sent aftur',
+  RECEIVE: ':eyes: Móttekið',
+  CLOSE_INDICTMENT: ':white_check_mark: Lokið',
   ACCEPT: ':white_check_mark: Krafa samþykkt',
   REJECT: ':negative_squared_cross_mark: Kröfu hafnað',
-  DELETE: ':fire: Krafa dregin til baka',
+  DELETE: ':fire: Afturkallað',
   SCHEDULE_COURT_DATE: ':timer_clock: Kröfu úthlutað fyrirtökutíma',
```
Comment (gudjong) [inline apps/judicial-system/backend/src/app/modules/event/event.service.ts:48]: This event can also happen for indictments. Prosecutors get COURT_DATE notifications for indictments and the event is triggered there.

File: libs/judicial-system/formatters/src/lib/formatters.ts [hunk 5]
```diff
@@ -115,7 +115,7 @@ export const caseTypes: CaseTypes = {
   SEXUAL_OFFENSES_OTHER_THAN_RAPE: 'kynferðisbrot önnur en nauðgun',
   MAJOR_ASSAULT: 'líkamsárás - meiriháttar',
   MINOR_ASSAULT: 'líkamsárás - minniháttar',
-  AGGRAVATED_ASSAULT: 'líkamsáras - sérlega hættuleg',
+  AGGRAVATED_ASSAULT: 'líkamsárás - sérlega hættuleg',
   ASSAULT_LEADING_TO_DEATH: 'líkamsáras sem leiðir til dauða',
```
Comment (gudjong) [inline libs/judicial-system/formatters/src/lib/formatters.ts:119]: Please also fix this one.

**Review by gudjong — CHANGES_REQUESTED** [review]
Some comments

**Review by Arnarkari93 — APPROVED** [review]

**Review by gudjong — APPROVED** [review]

**Pull request merged**